src/components/ResultsCount/ResultsCount.tsx

```diff
@@ -26,7 +38,30 @@ export const ResultsCount = ({
           {currentCount} results of {resultsCount}
         </strong>
       </p>
-      <p className="cc-results-count__sort">Sorted by {sortedBy}</p>
+      {sortedBy && (
```

Comment:
There is a possibility of sortedBy text and the `options` select being rendered at the same time. Is it worth having some logic here to decide which one is displayed by preference? Would make more sense to prefer the select I think if sort options are available.